src/logic/safe/store/models/types/transaction.ts

```diff
@@ -101,3 +106,13 @@ export type TxArgs = {
   to: string
   valueInWei: string
 }
+
+export type SafeModuleTransaction = ModuleTxServiceModel & {
+  nonce?: string // not required for this tx: added for compatibility
+  fee?: number // not required for this tx: added for compatibility
+  executionTxHash?: string // not required for this tx: added for compatibility
+  safeTxHash: string // table uses this key as a unique row identifier, added for compatibility
```

Comment:
Would make sense to "export" those compatibility types to another type and do something like `ModuleTxServiceModel & CompatibilityTypes`?